test/inductor/test_torchinductor.py

```diff
@@ -10571,56 +10571,44 @@ def fn(x, y, x_dtype, x2):
             x2 = x2.view(x_dtype) + 1
             return x @ y, x2 @ x
 
-        def _get_primitive_bitwidth(dtype):
-            if dtype.is_floating_point:
-                return torch.finfo(dtype).bits
-            else:
-                return torch.iinfo(dtype).bits
-
-        test_dtypes = [
-            torch.float32,
-            torch.float64,
-            torch.float16,
-            torch.bfloat16,
-            torch.uint8,
-            torch.int8,
-            torch.int16,
-            torch.int32,
-            torch.int64,
-        ]
-        for test_dtype_x in test_dtypes:
-            for test_dtype_y in test_dtypes:
-                # @ operation needs arguments to be the same dtype
-                for view_dtype in test_dtypes:
```

Comment:
I would keep the loop (it's simpler) and then test eager first to see if it failed and continue in that case. but this is fine

Reply:
good idea. fixed in https://github.com/pytorch/pytorch/pull/128883/commits/66d883a95005989d507f3a6f178e90cd3204be09